config/nextflow_config/singularity.config

```
@@ -20,10 +20,10 @@ process {
         container = 'https://depot.galaxyproject.org/singularity/illumina-interop:1.2.4--hdbdd923_2'
     }
     withName: 'MULTIQC_PER_FLOWCELL' {
-        container = 'https://depot.galaxyproject.org/singularity/multiqc:1.11--pyhdfd78af_0'
+        container = 'https://depot.galaxyproject.org/singularity/multiqc:1.20--pyhdfd78af_1'
```

```suggestion
        container = 'https://depot.galaxyproject.org/singularity/multiqc:1.21--pyhdfd78af_0'
```
A new version came out 5 day ago, sorry about that, we should really make an effort to review this faster and keep up with MultiQC releases :laughing:

---

@Aratz Thanks for that update. I have now updated to the latest version